aleo/caip2.md

```diff
@@ -0,0 +1,107 @@
+---
+namespace-identifier: aleo-caip2
+title: Aleo Network - Namespace Chains
+author: Jonathan Gonzalez (@jonandgon)
+discussions-to: <URL of PR, mailing list, etc>
+status: Draft
+type: Standard
+created: 2023-09-12
+requires (*optional): <["CAIP-2"]>
+replaces (*optional): <CAIP-2>
+---
+
+<!--You can leave these HTML comments in your merged CAIP and delete the 
+ visible duplicate text guides, they will not appear and may be helpful to 
+ refer to if you edit it again. This is the suggested template for new CAIPs.
+ Note that an CAIP number will be assigned by an editor. When opening a pull
+ request to submit your EIP, please use an abbreviated title in the 
+ filename, `caipX.md`, all lowercase, no `-` between the CAIP and its 
+ number.-->
+
+# CAIP-2
+
+*For context, see the [CAIP-2][https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-2.md] specification.*
+
+## Rationale
+
+The namespace `aleo` refers to the Aleo Network Layer 1 blockchain.
+
+Aleo consists of a single network: a testnet network (Testnet3).
+
+An identifier for a Aleo chain consists of the namespace prefix "aleo:"
+followed by the chain id.
+
+## Syntax
+
+The Aleo chain ID is the name of the chain. e.g. `testnet3` or `mainnet` when mainnet is released.
```

Comment:
It's hard to say right now since there is currently only one chain and there is no chain ID spec available.

For now, I would say `/^[a-z0-9]{1,15}$/`.